### DISCUSSION TITLE
Pull request template

### DISCUSSION THREAD

**Discussion by ihhub** [ihhub/fheroes2#5817] [Ideas]
We can add a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) for pull requests like we did for issues. I noticed that some pull requests lack any description which is hard to understand their purpose. Newcomers might find this template every useful and the team might not need to spend enough time to correct the pull request. Some of examples of the templates can be found [here](https://axolo.co/blog/p/part-3-github-pull-request-template). We might make separate types of pull requests depending on their content: translations, new features and bugs.

This is just a proposal so any ideas are welcome.

**Comment by oleg-derevenetz** [comment]
Hi @ihhub how do you see the PR template for yourself? So far, all that comes to mind is a mandatory description of the changes and a list of issues that may be closed by this PR (or related to it), it doesn't look enough for a template :)
